collect_app/src/main/java/org/odk/collect/android/activities/MainMenuActivity.java

```diff
@@ -574,31 +582,4 @@ private void displayBannerWithSuccessStorageMigrationResult() {
             storageMigrationRepository.clearResult();
         });
     }
-
-    private void importSettingsFromLegacyFiles() {
-        try {
-            String settings = new LegacySettingsFileReader(storagePathProvider).toJSON();
-
-            if (settings != null) {
-                String type = new File(storagePathProvider.getStorageRootDirPath() + "/collect.settings.json").exists()
```

Comment:
This actually wouldn't ever have worked as `LegacySettingsFileReader` deleted the files. I've moved that out to a separate action so this should now fire correctly.